recipes/sentry-crashpad/all/conanfile.py

```diff
@@ -95,8 +95,9 @@ def generate(self):
     def build(self):
         apply_conandata_patches(self)
         openssl_repl = "find_package(OpenSSL REQUIRED)" if self.options.get_safe("with_tls") else ""
-        replace_in_file(self, os.path.join(self.source_folder, "external", "crashpad", "CMakeLists.txt"),
-                        "find_package(OpenSSL)", openssl_repl)
+        if Version(self.version) < "0.6.1":
```

Comment:
This should ideally be a patch file

Reply:
The problem is that it depends on the `with_tls` option. Is there a way to apply a patch conditionally ?

Reply:
No! You are spot on